File: TaskManager/ClientApp/src/app/api/models/project-rm.ts
/* tslint:disable */
/* eslint-disable */
import { Tasks } from './tasks';
import { User } from './user';
export interface ProjectRm {
  projectDescription?: null | string;
  projectId?: string;
  projectName?: null | string;
  tasks?: null | Array<Tasks>;
  users?: null | Array<User>;
}

File: TaskManager/ReadModels/ProjectRm.cs
using TaskManager.Domain.Entities;

namespace TaskManager.ReadModels
{

	/*
		Leaving the ProjectRM with Users and Tasks for now
		to test the API. Might remove later
		 */
	public record ProjectRm(
		Guid ProjectId,
		string ProjectName,
		string? ProjectDescription,
		IList<User>? Users,
		IList<Tasks>? Tasks
		);
}



/*public record ProjectRm(
		Guid ProjectId,
		string ProjectName,
		string ProjectDescription,
		IList<UserRm> Users,
		IList<TasksRm> Tasks
		);*/